src/plugins/home/public/application/components/home_list_card.tsx

```diff
@@ -64,39 +64,50 @@ interface Config {
 }
 
 export const HomeListCard = ({ config }: { config: Config }) => {
+  const isMobile = useIsWithinBreakpoints(['xs', 's', 'm', 'l']);
+
+  const overflowStyle = {
+    overflow: isMobile ? 'scroll' : 'visible',
+    maxHeight: '230px',
+  };
+
   return (
     <>
-      <EuiPanel paddingSize="s" hasBorder={false} hasShadow={false}>
-        <EuiTitle>
-          <h4>{config.title}</h4>
-        </EuiTitle>
-        <EuiSpacer />
-        {config.list.length > 0 && (
-          <EuiDescriptionList>
-            {config.list.map((item) => (
-              <>
-                <EuiDescriptionListTitle>
-                  <EuiLink href={item.href} target="_blank">
-                    {item.label}
-                  </EuiLink>
-                </EuiDescriptionListTitle>
-                <EuiDescriptionListDescription>{item.description}</EuiDescriptionListDescription>
-              </>
-            ))}
-          </EuiDescriptionList>
-        )}
-
-        {config.allLink ? (
-          <>
-            <EuiSpacer />
-            <EuiLink href={config.allLink} target="_blank">
-              <EuiText size="s" style={{ display: 'inline' }}>
-                View all
-              </EuiText>
-            </EuiLink>
-          </>
-        ) : null}
-      </EuiPanel>
+      <EuiCard
+        textAlign="left"
+        title={
+          <EuiTitle>
+            <h4>{config.title}</h4>
+          </EuiTitle>
+        }
+        description={
+          config.list.length > 0 && (
+            <EuiDescriptionList style={overflowStyle}>
+              {config.list.map((item) => (
+                <>
+                  <EuiDescriptionListTitle>
+                    <EuiLink href={item.href} target="_blank">
+                      {item.label}
+                    </EuiLink>
+                  </EuiDescriptionListTitle>
+                  <EuiDescriptionListDescription>{item.description}</EuiDescriptionListDescription>
+                </>
+              ))}
+            </EuiDescriptionList>
+          )
+        }
+        footer={
+          config.allLink ? (
+            <>
+              <EuiLink href={config.allLink} target="_blank">
+                <EuiText size="s" style={{ display: 'inline' }}>
```

Comment:
We have some css utility, please try to avoid using css-inline style.
https://oui.opensearch.org/1.8/#/utilities/css-utility-classes

Reply:
Thanks, already updated